tests/dataframe_tests/test_data/parent/etlMapping.yaml

```diff
@@ -0,0 +1,27 @@
+mappings:
+  - name: qa-ibd
+    doc_type: participant
+    type: aggregator
+    root: participant
+    props:
+      - name: submitter_id
+      - name: project_id
+    parent_props:
```

Comment:
from the ticket description i understand that the change is for "parent_props". Can we remove the other props in this etl mapping?

Reply:
aggregated_props is required and props just have minimum props. It also does not effect the test.

Reply:
even if it doesn't affect the test, it's a lot easier to understand a unit test when there isn't unrelated data and we need to figure out what is relevant and what isn't

how about removing the `flatten_props`?

Reply:
yeah, I will remove `flatten_props`